molecule/resources/create.yml

```diff
@@ -0,0 +1,101 @@
+---
```

Comment:
Why do we need that?

Reply:
This driver is modified to create additional volume for swap.